pkg/products/monitoring/prometheusRules.go

```diff
@@ -282,6 +281,22 @@ func (r *Reconciler) newAlertsReconciler() resources.AlertReconciler {
 					},
 				},
 			},
+			{
+				AlertName: "general-monitoring-alerts",
+				Namespace: r.Config.GetOperatorNamespace(),
+				GroupName: "general.rules",
+				Rules: []monitoringv1.Rule{
+					{
+						Alert: "TargetDown",
+						Annotations: map[string]string{
+							"message": "{{ printf \"%.4g\" $value }}% of the {{ $labels.job }}/{{ $labels.service }} targets in {{ $labels.namespace }} namespace are down.",
```

Comment:
Remove the forward "/".
The namespace value is not displayed in the annotation of the alert.
Current message is shown as: 
```33.33% of the 3scale-apicast-pods/ targets in namespace are down.```